03LessonThree/homework/git-scm.html

```diff
@@ -3,58 +3,68 @@
-                <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place for beginners to ask about anything.</p>
-            </div>
-        </div>
-        <div id="aside">
+                <p>You do not need to subscribe: you will be Cc'd in replies. Please keep the Cc list intact when replying (use "Reply to all"). Greylisting may delay your first post for a few hours. Note also that the list only accepts plain-text email;
+                    please disable HTML in your outgoing messages.</p>
+                <p>The archive can be found on Gmane. Click here to subscribe. If you have trouble sending to the list with your regular mail reader, you may also post via gmane. There is also Git user mailing list on Google Groups which is a nice place
+                    for beginners to ask about anything.</p>
+            </article>
+        </section>
+        <aside>
             <div>
-            <h4>Rapid development with FTS</h4>
-            <p>An enterprise-grade server for teams to share code, track work, and ship software — for any language, all in a single package.
-It’s the perfect complement to your IDE.</p>
-            <img src="http://www.discountasp.net/tfs/images/tfs-2012-metro.png">
+                <h4>Rapid development with FTS</h4>
+                <p>An enterprise-grade server for teams to share code, track work, and ship software — for any language, all in a single package. It’s the perfect complement to your IDE.</p>
+                <img src="http://www.discountasp.net/tfs/images/tfs-2012-metro.png">
             </div>
             <div>
-            <h4>Node.js®</h4>
-            <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.</p>
-            <img src="http://nodejs-cloud.com/img/128px/nodejs.png">
+                <h4>Node.js®</h4>
+                <p>It is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries
+                    in the world.</p>
+                <img src="http://nodejs-cloud.com/img/128px/nodejs.png">
             </div>
-        </div>
-    </div>
-    <div id="footer">
-        This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
-    </div>
+        </aside>
+    </main>
+    <footer>
+        This open sourced site is hosted on GitHub.
+        <br/> Patches, suggestions and comments are welcome.
+    </footer>
+</body>
+
+</html>
+te is hosted on GitHub.

```

Comment:
It looks like you accidentally pasted in html markup after the closing `</html>`. There should not be any html markup here. Good job otherwise, it was almost perfect. I'm sure you this will not happen again. :smiley: